com.unity.render-pipelines.high-definition/Runtime/Material/Decal/DecalData.hlsl

```diff
@@ -31,7 +31,17 @@ void GetSurfaceData(FragInputs input, float3 V, PositionInputs posInput, float a
 #ifdef _MATERIAL_AFFECTS_EMISSION
     surfaceData.emissive = _EmissiveColor.rgb * fadeFactor;
     #ifdef _EMISSIVEMAP
-    surfaceData.emissive *= SAMPLE_TEXTURE2D(_EmissiveColorMap, sampler_EmissiveColorMap, texCoords).rgb;
+    float2 emissiveColorMapSize;
+    float emissiveColorMapLODs;
+    _EmissiveColorMap.GetDimensions(0, emissiveColorMapSize.x, emissiveColorMapSize.y, emissiveColorMapLODs);
```

Comment:
Hey, so GetDimensions isn't correctly supported on various platform. To retrieve the number of LOD you should use our function uint GetMipCount(TEXTURE2D_PARAM(tex, smp)) declare in common.hlsl. It isn't implemented for all platform but return 0 at least. Which in your context is not ideal but should be ok (in particular for Metal). Otherwise you need to do it a different way without trying to access to current number of LOD or by "fixing" it.

However for the size of the texture itself I have no idea if we have something cross platforms. From the comment of the function GetMipCount, it is possible that retrieving the size works. If we want to validate all this, best is to create a test scene with the artifact and add it to our project suite, then run it on Yamato to see what is happening on author platforms.

Reply:
There's also an issue I haven't yet figured out, where the background that's not rendered to G-buffer has some emissive dots in the silhouette. It's less of an issue than before, but it's  something I need to investigate